### ISSUE TITLE
[Deque Analysis] Disabled Side Menu options have insufficient color contrast

### ISSUE THREAD

**Issue by tdonohue** [DSpace/dspace-angular#1155]
## Deque Analysis Summary

Options which appear in the Admin/Side menu but are *disabled* have insufficient color contrast.

These have two separate errors (as there are slightly different colors for disabled headers vs sublinks)

1. Element has insufficient color contrast of 2.25 (foreground color: #4b545e, background color: #0f1b28
     * Deque issue ticket IDs: 469232, 469233, 469234, 469235, 469236, 469237
2. Element has insufficient color contrast of 2.26 (foreground color: #4e5a66, background color: #132333
     * Deque issue ticket IDs: 469238, 469239 

## Recommended Fix

These will be fixed by #1145 by removal of these disabled links.  Disabled links are no longer supported in this sidemenu. If you don't have access to the link, it is hidden.
